packages/langchain/src/openai/chat.ts

```diff
@@ -72,6 +73,19 @@ export class AzureOpenAiChatClient extends BaseChatModel<AzureOpenAiChatCallOpti
     this.presence_penalty = fields.presence_penalty;
     this.frequency_penalty = fields.frequency_penalty;
     this.max_tokens = fields.max_tokens;
+    // Initialize streaming flags with LangChain-compatible behavior:
+    // - `streaming`: true enables auto-streaming in `invoke()` calls
+    // - `disableStreaming`: true overrides streaming flag
+    // - `streaming`: `false` causes `disableStreaming` to be set to `true` for framework compatibility
+    this.disableStreaming = fields.disableStreaming ?? this.disableStreaming;
+    // if streaming is explicitly false, streaming is disabled
+    if (fields?.streaming === false) {
+      this.disableStreaming = true;
+    }
+    // Enable streaming only when `streaming` is `true` (default `false`) and `disableStreaming` is not `true` (default `undefined`).
+    this.streaming =
+      (fields?.streaming ?? this.streaming) && this.disableStreaming !== true;
```

Comment:
[pp] Here similarly, truthy value of `fields?.streaming` might be a problem in JS